introspector/src/core/telemetry_subscription.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+// This file is part of polkadot-introspector.
+//
+// polkadot-introspector is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// polkadot-introspector is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with polkadot-introspector.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+use super::TelemetryFeed;
+use color_eyre::Report;
+use futures_util::StreamExt;
+use log::info;
+use tokio::{
+	net::TcpStream,
+	sync::{
+		broadcast::Sender as BroadcastSender,
+		mpsc::{error::SendError, Sender},
+	},
+};
+use tokio_tungstenite::{connect_async, tungstenite::Message, MaybeTlsStream, WebSocketStream};
```

Comment:
Why don't we use `jsonrpsee` as `subxt` does?

Reply:
It seems we cannot, and the `tokio_tungstenite` is a de-facto standard when you just need to subscribe on events through a websocket.